pymc/ode/utils.py

```diff
@@ -103,10 +103,17 @@ def augment_system(ode_func, n_states, n_theta):
 
     # Get symbolic representation of the ODEs by passing tensors for y, t and theta
     yhat = ode_func(t_y, t_t, t_p[n_states:])
-    # Stack the results of the ode_func into a single tensor variable
-    if not isinstance(yhat, (list, tuple)):
-        yhat = (yhat,)
-    t_yhat = at.stack(yhat, axis=0)
+    if isinstance(yhat, at.TensorVariable):
+        t_yhat = at.atleast_1d(yhat)
+        if t_yhat.ndim > 1:
```

Comment:
Does this check also make sense in the `else` branch? If so, you could move it below the if/else statements.

Reply:
I moved the check so that it also applies to the else branch.

I also created two tests which check that the `ValueError` is thrown.